src/Service.Tests/dab-config.CosmosDb_NoSql.json

```diff
@@ -40,9 +37,7 @@
     "Planet": {
   "source": {
     "object": "graphqldb.planet",
-    "type": "table",
-    "parameters": null,
-    "key-fields": null
+    "type": "table"
```

Comment:
this is interesting, the `type` should not be here since its cosmosdb_nosql, besides `table` doesnt make sense.

Reply:
Could we get away with making a new `EntitySourceType` of `Collection` that is for CosmosDB NoSQL?

I worry that it's so heavily relied on as a non-nullable property in SQL that we'd spend a lot of time having to either use `!` to ignore the null check, or writing null checks for minimal value.

This is one of the use cases that surfaced when doing the new config engine that there are very tiny discrepancies between how SQL and Cosmos work and that we've kind of just lived with having to do constant type checking.

Reply:
#1530 addressed this